cloud-sql/r2dbc/src/main/java/com/example/cloudsql/r2dbcsample/CommandLineRunnerSchemaCreator.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloudsql.r2dbcsample;
+
+import io.r2dbc.spi.ConnectionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.data.r2dbc.core.DatabaseClient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CommandLineRunnerSchemaCreator implements CommandLineRunner {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(CommandLineRunnerSchemaCreator.class);
+
+  @Autowired
+  private ConnectionFactory connectionFactory;
+
+  @Override
+  public void run(String... args) {
+    runDdl("CREATE TABLE IF NOT EXISTS vote ( "
+        + "vote_id SERIAL NOT NULL, "
+        + "time_cast timestamp NOT NULL, "
+        + "candidate CHAR(6) NOT NULL, "
+        + "PRIMARY KEY (vote_id) );");
+  }
```

Comment:
We've been doing separate samples for each DB engine, mostly because the SQL syntax can vary. This might not matter much between MySQL/Postgres, but for SQL Server, this command wouldn't work. We might want to split into separate samples for Postgres/MySQL for that reason.

Reply:
@shubha-rajan 
I would prefer to avoid code duplication, since this is the only thing that would be different. I think we could make it a variable and let users pass it to the application or put it into a config. What do you think?

Reply:
We can handle this on our end, but the samples will need to be separated out at some point if we want to link to these from Cloud Docs.